detection_node:
  ros__parameters:
    model_path: ""  # 여기에 best.pt 경로 넣어주세요
    rgb_topic: "/zedm/zed_node/rgb/color/rect/image"
    device: "0"              # (권장) "0" 또는 "cpu"  /  cuda:0 대신
    imgsz: 640
    half: true               # FP16 사용
    show_imshow: false       # imshow 디버그
    publish_debug: true      # rqt 디버그 이미지 토픽 publish
    debug_topic: "/debug_image"
    debug_rate_hz: 15.0

refiner_node:
  ros__parameters:
    half_win: 1  # 얼마나 주변 depth를 볼 것인가
    remove_space_dis: 10000  # mm

pantilt_node:
  ros__parameters:
    rate_hz: 50.0

    pan_id: 22
    pan_max_deg: 70.0
    pan_min_deg: -70.0

    tilt_id: 23
    tilt_max_deg: -5.0
    tilt_min_deg: -45.0

    scan_pan_speed_deg_s: 40.0
    scan_tilt_speed_deg_s: 40.0

    img_w: 640
    img_h: 360

    img_cut: 4

    pan_dir: -1.0
    tilt_dir: 1.0
